core/src/main/java/edu/wpi/grip/core/FileManager.java

```diff
@@ -0,0 +1,15 @@
+package edu.wpi.grip.core;
+
+/**
+ * Defines what methods a FileManager implementation must implement.
+ */
```

Comment:
That's the definition of an interface. What is a FileManager though? That's what you are trying to explain here.